main.go

```diff
@@ -86,6 +86,13 @@ func checkTaskFiles(t task) (checkResult, error) {
 }
 
 func main() {
+	// Looks for '-check' flag to specify checking files
+	check := false
```

Comment:
There is actually a standard library to handle flags nicely: https://golang.org/pkg/flag/